include/exadg/operators/operator_base.h

```diff
@@ -270,13 +270,16 @@ class OperatorBase : public dealii::Subscriptor
 
     if(dof_handler.has_level_dofs())
     {
```

Comment:
this is actually covered in
#713 but was needed to run.

Reply:
has been merged, so you could rebase?